x/fee/client/cli/query.go

```diff
@@ -107,7 +107,7 @@ func GetNonFungibleTokenFeeSetting(cdc *codec.Codec) *cobra.Command {
 
 			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "nonFungible", nonfungible.QueryTokenData, symbol), nil)
 			if err != nil {
-				fmt.Printf("Could not get non-fungible token fee setting: %s\n", err)
+				fmt.Printf("This non-fungible token does not exist\n")
```

Comment:
No nid this, please add in the symbol that user keyed in.

Reply:
@githubckgoh1439

Reply:
Updated the Error message along with enter-symbol.

Reply:
@githubckgoh1439 remove this please.

Reply:
Updated the Error message along with enter-symbol for non-fungible token.

Reply:
This updated:
https://github.com/maxonrow/maxonrow-go/pull/200/commits/e45c12e9b7a091d4e75d53460bbb632df252f9f0

Reply:
Fixed under this: 
https://github.com/maxonrow/maxonrow-go/pull/200/commits/28a37e076e1f52dda17d3bf53755b2ab281bc3e7